pkg/plugins/common/kustomize/v2-alpha/scaffolds/internal/templates/config/samples/crd_sample.go

```diff
@@ -53,6 +54,12 @@ func (f *CRDSample) SetTemplateDefaults() error {
 const crdSampleTemplate = `apiVersion: {{ .Resource.QualifiedGroup }}/{{ .Resource.Version }}
 kind: {{ .Resource.Kind }}
 metadata:
+  labels:
+    app.kubernetes.io/name: {{ lower .Resource.Kind }}
+    app.kubernetes.io/instance: {{ lower .Resource.Kind }}-sample
+    app.kubernetes.io/part-of: {{ .ProjectName }}
+    app.kuberentes.io/managed-by: {{ .ProjectName }}
```

Comment:
```suggestion
    app.kuberentes.io/managed-by: kustomize
```

Reply:
resolved in commit: https://github.com/kubernetes-sigs/kubebuilder/pull/2800/commits/de5ad3beb0988a49f533ee8c1a8c87fd82e7bf02